### PR TITLE
Make slight corrections for the docs branch.

### PR DESCRIPTION
**Pull request by Kichura**
Resolves a few grammar errors in the docs and adds some other touches here-and-there because why not?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# VanillaInstaller
+# Vanilla Installer
 
 <img alt="Built with Markdown" height="40" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@2/assets/compact/built-with/markdown_vector.svg">
 
-Welcome to the VanillaInstaller docs! In here, you can find useful information about this project and its development.
+Welcome to the Vanilla Installer docs! In here, you can find useful information about this project and it's development.
```
Comment (Madis0) [inline README.md]: "its" was correct here

File: credits.md [hunk 2]
```diff
@@ -275,6 +275,6 @@ SOFTWARE.
 
 * [https://docs.python.org/3.10/library/sys.html#sys.platform](https://docs.python.org/3.10/library/sys.html#sys.platform)
 
-### Other
+### Others
```
Comment (Madis0) [inline credits.md:278]: both are valid IMO

File: installation.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ It's recommended to install Python using [Homebrew](https://brew.sh), however in
 
 Because of Linux having many different package managers, we've decided to make the way to install on Linux via `pip`, or building from source using `python` and `poetry`, which isn't covered here.
 
-1. Install Python 3.8 or higher (3.11 recommended) from your package manager - for example `sudo apt install python3` or `sudo pacman -S python3`
+1. Install Python 3.8 or higher (3.11 recommended) from your package manager - for example `sudo apt install python3` or `sudo pacman -S python3` or `sudo dnf install python3`
```
Comment (Madis0) [inline installation.md]: What's that?

**Review by Madis0 — CHANGES_REQUESTED** [review]

File: installation.md (hunk 3 above)
Comment (Kichura) [reply installation.md]: This is fedora's default packaging command for installing python3.

File: credits.md (hunk 2 above)
Comment (Kichura) [reply credits.md:278]: I chose "others" because the wording made morely sense.

File: README.md (hunk 1 above)
Comment (Kichura) [reply README.md]: Darn.

File: installation.md (hunk 3 above)
Comment (Madis0) [reply installation.md]: Well then put the distro names in parentheses (to make it useful) or remove the last two examples altogether (to only have Debian's serve as an example).

Comment (Kichura) [reply installation.md]: On it.

Comment (Kichura) [reply installation.md]: Not the best i could do in commit but feel free to send suggestions on how to make it better if needed.

File: changelogs/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Changelogs
 
-The changelogs for VanillaInstaller follow a similar format to [Iris](https://irisshaders.net)'s format, with a full changelog and a summary.
+The changelogs for Vanilla Installer follows a similar format to [Iris](https://irisshaders.net)' format, with a full changelog and a summary.
```
Comment (Madis0) [inline changelogs/README.md]: changelogs follow or changelog follows

Comment (Kichura) [reply changelogs/README.md]: *(ahem).*

**Review by Madis0 — APPROVED** [review]
LGTM, though I leave the "Vanilla Installer"/"VanillaInstaller" naming decision to osfan

File: for-devs.md [hunk 5]
```diff
@@ -27,10 +27,12 @@ This one might be obvious, but please keep in mind to ALWAYS `Ctrl+F` before ren
 
 ## Tested on...
 
-- Linux → Debian → Ubuntu → PopOS:
-  - My main system, should work 100% fine
+- Debian → Ubuntu → Pop! OS:
```
Comment (osfanbuff63) [inline for-devs.md:30]: should these have spaces next to the arrows? or is it just github mobile formatting

**Review by osfanbuff63 — COMMENTED** [review]
Looks pretty good, just one quick comment

Honestly I think Vanilla Installer with the space looks a bit better, and it's used other places so this is the right move imo

File: for-devs.md (hunk 5 above)
Comment (Kichura) [reply for-devs.md:30]: They use periods to let you figure out if there is a space there or not, In this case; It's intentional accessibility which i did not purposely add.

**Review by osfanbuff63 — APPROVED** [review]
LGTM!

**Pull request merged**